config/nginx.conf

```diff
@@ -9,14 +9,8 @@ server {
     keepalive_timeout 5;
     client_max_body_size 1G; # Was 4G
 
-    # managed by Certbot
-    ssl_certificate /etc/letsencrypt/live/api.sfucsss.org/fullchain.pem;
```

Comment:
I'm assuming these lines got moved somewhat that is managed by certbot automatically, could you tell me a bit more about how the new certbot setup will work?

Reply:
Yes; now, the preferred server configuration is listed under an HTTP server (`listen 80;`), which is nginx-valid (it doesn't use SSL), but that certbot adapts to *become* an HTTPS server (`listen 443;`). This keeps the configs on the repo super simple and leaves much of the SSL configuration to how certbot handles it :)